README.md

```diff
@@ -20,102 +22,25 @@ The crux of the workflow will be based on facilitating
 1. **goal setting** and
 1. **quality control**
 
-(Balkundi and Harrison, 2006).
+([Balkundi and Harrison, 2006](https://doi.org/10.5465/amj.2006.20785500)).
 Quality assurance will come from code reviews initiated by pull requests.
-A goal will be complete when a pull request is merged into the main branch.
-Details provided in the workflow will help individuals
-communicate,
-document, and
-engage
-while collaborating,
-which will 
-increase transparency and awareness while reducing the overhead needed for communication.
-For example,
-Christine's efforts to ensure meeting notes are available in a document on google drive for each meeting
-reduces overhead because people know exactly where to look for these notes and do not have to email Christine.
+Details provided in the workflow will help individuals communicate, document,
+and engage while collaborating, which will increase transparency and awareness
+while reducing the overhead needed for communication.
 
 The highest priority is to develop a workflow outlining
-**best practices for pull requests**, i.e., GitHub workflow.
-
-## Best practices
-
-The complete workflow includes information on
-
-  * **GitHub workflow**: all options stress code reviews as best practice.
-
-    * Available options
-
-      * centralized:
-      * forks:
-      * branches:
-      * branches with main and development:
-      * gitflow
-
-    * Teams
-
-      * Who can contribute code == anyone
-      * Who can review code
-      * Who can merge into main branch
-      * Sub-team size (suggestions of 4-5)
-
-    * Commit size
-
-  * Git commit message
-
-    * what it means to be human readable
-    * subject line
-    * additional information
-
-  * GitHub issues
-
-    * issue templates
-    * labels
-
-  * [Context switching in git](https://opensource.com/article/21/4/context-switching-git)
-
-    * stash + branch
-    * WIP commit + branch
-    * new clone
-    * worktree
-
-  * Code style guide
-  * Working directory
-
-    * [Don't use `setwd()` within R by Jenny Bryan](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/)
-    * Relative paths with standard folder structure
-
-## Potential problems
-
-  * updating issues requires effort;
-  * lack of communication infrastructure for non-coders that do not already know GitHub;
-  * merge conflicts because originating from a lack of knowledge on what others are working on;
-  * insufficient options for pointed discussions between a small number of team members,
-    individuals tend to migrate to instant messaging or email to work things out rather then keeping conversations on GitHub;
-  * centralized and branch workflows require trust;
+**best practices for [pull requests](https://help.github.com/articles/using-pull-requests)**,
+i.e., GitHub workflow.

```

Comment:
Do we need to highlight "GitHub workflow" here? For some people, 'GitHub workflow' may specifically refer to GitHub Actions workflows defined by a YAML file, This is also the first search result I got on google.

Reply:
I think we can just delete line 33 and end line 32 with a full stop. How does that sound. Or, if you want you can suggest text for what you were thinking.

Reply:
Deleting line 33 sounds good to me!